### ISSUE TITLE
doubt

### ISSUE THREAD

**Issue by Mau-dvr75** [RUseismology/ANIMATIVITY#2]
Is it possible to calculate the response for a layered isotropic media?

**Comment by RU21Seismology** [comment]
Yes, you can set the some part of isotropic media as 0 to make it isotropic. For instance, if you use Backus Notation, A=lambda + 2mu, D = mu, B=C=E=0, that is treating A and D as Lame parameters. If you use the 6 by 6 matrix, just input the isotropic matrix you have and it will work. In general, the code does not know if it is isotropic or not, as long as the input is valid you will get the synthesis.
